pkg/controller/pipelines.go

```diff
@@ -42,10 +34,11 @@ func (c *Controller) PullRefMetrics(ctx context.Context, ref schemas.Ref) error
 	}
 
 	pipelines, _, err := c.Gitlab.GetProjectPipelines(ctx, ref.Project.Name, &goGitlab.ListProjectPipelinesOptions{
-		// We only need the most recent pipeline
 		ListOptions: goGitlab.ListOptions{
-			PerPage: 1,
+			PerPage: int(ref.Project.Pull.Pipeline.PerRef),
 			Page:    1,
+			OrderBy: "updated_at",
```

Comment:
These two are new. Should this be removed or made configurable?

Reply:
True, removed it.